src/ert/_c_wrappers/job_queue/ext_job.py

```diff
@@ -60,24 +57,24 @@ class ExtJob(BaseCClass):
     )
     _get_arglist = ResPrototype("stringlist_ref ext_job_get_arglist(ext_job)")
     _set_arglist = ResPrototype("void ext_job_set_args(ext_job, stringlist)")
-    _get_argvalues = ResPrototype("stringlist_ref ext_job_get_argvalues(ext_job)")
     _clear_environment = ResPrototype("void ext_job_clear_environment(ext_job)")
     _save = ResPrototype("void ext_job_save(ext_job)")
-    _get_private_args = ResPrototype("subst_list_ref ext_job_get_private_args(ext_job)")
-    _set_define_args = ResPrototype(
-        "subst_list_ref ext_job_set_define_args(ext_job, subst_list)"
-    )
     _free_deprecated_argv = ResPrototype("void ext_job_free_deprecated_argv(ext_job)")
 
-    _copy = ResPrototype("ext_job_obj ext_job_alloc_copy(ext_job)")
-
     def __init__(
         self,
         config_file: str,
         private: bool,
         name: Optional[str] = None,
         search_PATH: bool = True,
     ):
+        self._name = name
+        self._config_file = config_file
+        self._private = private
```

Comment:
i like `self. _is_private` better as i reads better.... But leave it up to you... Just a very very minor comment :sweat_smile:

Reply:
Good suggestion!